### ISSUE TITLE
gold-cc-input accessibility issues 

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/gold-cc-input#10]
Audit results: 

[Severe] Controls and media elements should have labels (6)
[Warning] Text elements should have a reasonable contrast ratio (4)
[Warning] These elements are focusable but either invisible or obscured by another element (1)
Not applicable tests (9)
Passing tests (5)

Contrast: hint text and the credit card names are too light. 

Firefox / NVDA / Windows: hint text reads "autocomplete", but is that actually the case? Just wanted to check.

**Comment by notwaldorf** [comment]
I can't get NVDA (on IE10) to read "autocomplete". When I tab to an input, it either reads "edit, blank", if it's empty, or the credit card number, if it's filled. Am I using it wrong?

**Comment by notwaldorf** [comment]
Done! :tada:
